fe1-web/src/features/rollCall/components/RollCallOpen.tsx

```diff
@@ -0,0 +1,175 @@
+import { CompositeScreenProps, useNavigation } from '@react-navigation/core';
+import { StackScreenProps } from '@react-navigation/stack';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { Text } from 'react-native';
+import { useToast } from 'react-native-toast-notifications';
+import ReactTimeago from 'react-timeago';
+
+import { CollapsibleContainer, QRCode } from 'core/components';
+import ScreenWrapper from 'core/components/ScreenWrapper';
+import { ToolbarItem } from 'core/components/Toolbar';
+import { AppParamList } from 'core/navigation/typing/AppParamList';
+import { LaoEventsParamList } from 'core/navigation/typing/LaoEventsParamList';
+import { LaoParamList } from 'core/navigation/typing/LaoParamList';
+import { Hash, PublicKey, Timestamp } from 'core/objects';
+import { ScannablePopToken } from 'core/objects/ScannablePopToken';
+import { Typography } from 'core/styles';
+import { FOUR_SECONDS } from 'resources/const';
+import STRINGS from 'resources/strings';
+
+import { RollCallHooks } from '../hooks';
+import { requestCloseRollCall } from '../network';
+import { RollCall } from '../objects';
+import AttendeeList from './AttendeeList';
+
+type NavigationProps = CompositeScreenProps<
+  StackScreenProps<LaoEventsParamList, typeof STRINGS.navigation_lao_events_view_single_roll_call>,
+  CompositeScreenProps<
+    StackScreenProps<LaoParamList, typeof STRINGS.navigation_lao_events>,
+    StackScreenProps<AppParamList, typeof STRINGS.navigation_app_lao>
+  >
+>;
+
+const RollCallOpen = ({ rollCall, laoId, isOrganizer, scannedPopTokens }: IPropTypes) => {
+  const generateToken = RollCallHooks.useGenerateToken();
+  const hasSeed = RollCallHooks.useHasSeed();
+  const toast = useToast();
+  const navigation = useNavigation<NavigationProps['navigation']>();
+
+  const [popToken, setPopToken] = useState('');
+  const [hasWalletBeenInitialized, setHasWalletBeenInitialized] = useState(hasSeed());
+
+  const onAddAttendees = useCallback(() => {
+    // Once the roll call is opened the first time, idAlias is defined
+    if (rollCall.idAlias) {
+      navigation.navigate(STRINGS.navigation_app_lao, {
+        screen: STRINGS.navigation_lao_events,
+        params: {
+          screen: STRINGS.navigation_lao_events_open_roll_call,
+          params: {
+            rollCallId: rollCall.id.toString(),
+            attendeePopTokens: (scannedPopTokens || []).map((e) => e.valueOf()),
+          },
+        },
+      });
+    } else {
+      toast.show(STRINGS.roll_call_location_error_scanning_no_alias, {
+        type: 'danger',
+        placement: 'bottom',
+        duration: FOUR_SECONDS,
+      });
+      console.debug(STRINGS.roll_call_location_error_scanning_no_alias);
+    }
```

Comment:
The else statement is pretty much a (disguised) guard clause. 
You might want to structure this code as such.
```suggestion
    if (!rollCall.idAlias) {
      toast.show(STRINGS.roll_call_location_error_scanning_no_alias, {
        type: 'danger',
        placement: 'bottom',
        duration: FOUR_SECONDS,
      });
      console.warn(STRINGS.roll_call_location_error_scanning_no_alias);
      return;
    }
    
    navigation.navigate(STRINGS.navigation_app_lao, {
      screen: STRINGS.navigation_lao_events,
      params: {
        screen: STRINGS.navigation_lao_events_open_roll_call,
        params: {
          rollCallId: rollCall.id.toString(),
          attendeePopTokens: (scannedPopTokens || []).map((e) => e.valueOf()),
        },
      },
    });
```

Reply:
I agree!